src/sentry/integrations/vercel/client.py

```diff
@@ -45,4 +45,18 @@ def create_deploy_webhook(self):
             "events": ["deployment"],
         }
         response = self.post(self.WEBHOOK_URL, data=data)
+    def get_env_vars(self, vercel_project_id):
+        return self.get(self.GET_ENV_VAR_URL % vercel_project_id)
+
+    def get_secret(self, name):
+        return self.get(self.GET_SECRET_URL % name.lower())["uid"]
+
+    def create_secret(self, vercel_project_id, name, value):
+        data = {"name": name, "value": value}
```

Comment:
@ceorourke Do you think it make sense to always use the lower case value for the name? Just wondering.

Reply:
I don't think it matters much whether they key we use to create the secret is lowercase or uppercase, I did this because when checking if the secret already exists yet or not, if I didn't lowercase it then I was getting a false positive (that it doesn't exist) and then it error'ed out when attempting to create a secret that already existed.